docs/scenarios/library-authoring/fastening-libraries-to-rc2.md

```diff
@@ -0,0 +1,70 @@
+# How to manage package dependency versions for RC2
+
+By [Phillip Carter](https://github.com/cartermp)
+
+This article covers what you need to know about package versions for authoring libraries.
+
+## Glossary
+
+**Metapackage** - A NuGet package that represents a set of NuGet packages.  They're organized in a way that makes sense for general library and app development.
+
+**Trimming** - The act of removing the packages you do not depend on from a metapackage.  The end result is that your library only depends on packages it uses.  See guidance in [How to trim your package dependencies](trimming.md).
+
+## Fasten your dependencies to RC2
+
+When creating libraries with .NET Core RC2, it's important that you fasten your dependencies to RC2.  This means two things.  First, you should only depend on packages marked as final for .NET Core RC2.  Second, each package should come from the same build. This means every package should have this pattern: `<package_version>-rc2-<build_number>`.
+
+**Note: The build number at the end is not final right now.  It will have a final number when RC2 ships.**
+
+**Examples of packages fastened to RC2**
+
+`"System.Collections":"4.0.11-rc2-24027"`
+
+`"NETStandard.Library":"1.5.0-rc2-24027"`
+
+`"Microsoft.NETCore.App":"1.0.0-rc2-24027"`
+
+**Examples of packages that are NOT fastened to RC2**
+
+`"System.Collections":"4.0.11-rc3-24021-00"`
+
+`"System.Net.Http":"4.1.0-rc2-24008"`
+
+`"System.Text.RegularExpressions":"4.0.10-rc3-24021-00"`
+
+### Why does this matter?
+
+We guarantee that if you fastened your dependencies to RC2, those packages will all work with together.  There is no such guarantee if you use packages which aren't fastened to RC2.
+
+### Is there some big list to look at with version numbers and stuff?
```

Comment:
Seems like an odd question and very informal using stuff, etc.
Maybe something like: Is there a central place where I can see these version numbers?